src/test/resources/application-test.yml

```diff
@@ -20,3 +20,6 @@ spring:
   jpa:
     show-sql: true
 
+  test:
+    database:
+      replace: none
```

Comment:
default is to replace the database with an embedded one that then doesn't have the `Oracle` mode